chemie/events/templates/events/social/deregister_or_edit.html

```diff
@@ -28,10 +28,14 @@ <h5>
       Du er påmeldt.
     {% elif registration.status == 2 %}
       Du er på ventelisten og er nr. {{ queue_position }} i køen
-    {% elif registration.status == 3 %}
+    {% elif not allowed_grade %}
       Det er ikke åpent for ditt klassetrinn, men vi har notert din interesse.<br>
       Du blir påmeldt automatisk og tilsendt en e-post dersom dette endres.
     {% endif %}
+    {% if not allowed_group %}
+      Du er ikke kohortsjef. Hvis du ønsker å delta på dette arrangementet,
+      må du be kohortsjefen din melde deg på.
+    {% endif %}
```

Comment:
We should open an issue for after kohortpåmelding so we remember to make this message more general. E.g "Du er ikke medlem i en gruppe som kan melde seg på." etc.

Reply:
Agreed, i simply hardcoded it for this purpose.